raiden-ts/src/raiden.ts

```diff
@@ -193,6 +197,7 @@ export class Raiden {
     this.state$ = latest$.pipe(pluckDistinct('state'));
     // pipe action, skipping cached
     this.action$ = latest$.pipe(pluckDistinct('action'), skip(1));
+    this.blockTime$ = latest$.pipe(pluckDistinct('blockNumber'));
```

Comment:
I don't get this. Shouldn't this be `blockTime`?

Reply:
Woosh.. good catch :see_no_evil:

Reply:
It's weird that this wasn't catched by the type system.. I think `pluck` type inference evaluates to `unknown`, and then that draws the type from the declared `blockTime$: Observable<number>` member, narrowing the inference, while it should better be narrowed as `never`.. I may look into it later, but already queued a fix.. Will push with a rebase after the end of your review